README.md

```diff
@@ -31,9 +31,9 @@ We need to have all these sessions backed up somewhere if we do not want to lose
 
 The following are required before you can enable Hazelcast Session Clustering:
 
--   Target application or web server should support Java 1.8 or higher.
+-   Target application or web server should support Java 17 or higher.
 
--   Target application or web server should support Servlet 4.0 or higher spec. Note that this library supports, "javax" namespace, not the new "jakarta" one. 
+-   Target application or web server should support Servlet 6.1 or higher spec. Note that this library supports, "jakarta" namespace. 
```

Comment:
Nit - if it _exclusively_ supports `jakarta`, it should explain it doesn't support `javax` (and perhaps reference the last version that did).

Reply:
Hmm, should not we support tomcat 10 as well? Tomcat 10 supports servlet 6.0 and java 11 and higher. 

source: https://tomcat.apache.org/whichversion.html

Reply:
We are saying that we support Servlet 6.1. Servlet 6.1 can be implemented by tomcat, jetty or any other servlet container.